pkg/telefork/client.go

```diff
@@ -0,0 +1,110 @@
+package telefork
```

Comment:
Can we add even a really basic test here? Just a quicker drive by one? I feel like it sets things up better in case more changes are made in the future.

Reply:
Done